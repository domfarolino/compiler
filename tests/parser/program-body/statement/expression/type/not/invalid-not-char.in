program dominic is

  bool target;

begin

  target := not 'd';

end program.
