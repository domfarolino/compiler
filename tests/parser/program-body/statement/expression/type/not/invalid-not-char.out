[1;31mLine 7: [Expression]: operator 'not' cannot be affixed to chars; must only be applied to boolean-equivalent types
Line 7: Error parsing program body
[0m